lib/synced/strategies/full.rb

```diff
@@ -93,6 +90,10 @@ def perform
         end
       end
 
+      def reset_synced
+        # this is not supported by this strategy
```

Comment:
Hmm, actually `reset_synced` is called from synchronizer, not inside strategy, in such case might be better to leave this method and raise NotImplementedError or maybe RuntimeError,